tests/test_treeview/test_all_treeview.py

```diff
@@ -6,13 +6,13 @@
 ETEPATH = os.path.abspath(os.path.split(os.path.realpath(__file__))[0]+'/../')
 sys.path.insert(0, ETEPATH)
 
-from ... import Tree, TreeStyle, NodeStyle, PhyloTree, faces, random_color
```

Comment:
@jordibc this test runs fine, but I switched to absolute imports (which should be replaced all over the code). It should be clean up a bit an integrated into the pytest units. If the test runs OK, it generates an image in png,pdf and svg format called (`test.xxx`), which should contain something like the one attached 

![test](https://user-images.githubusercontent.com/163658/235622968-1a9f9ff5-4645-484b-87f4-a3ee8d8860b2.png)